CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Operations
 
+- Removed unassigned variable names
```

Comment:
Just one small thing, can you move this into the `Fixed` section, `Operations` are usually for the CI related stuff. Thanks!